src/atoms/InputField/inputfield.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../styles/variables.scss";
+
+.input-wrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border: 1px solid $gray-3;
+  border-radius: 8px;
+  &.success {
+    border: 1px solid $success;
+  }
+  &.warning {
+    border: 1px solid $warning;
+  }
+  &.error {
+    border: 1px solid $error;
+  }
+  svg {
+    color: $gray-4;
+    font-size: 2rem;
+    line-height: 2rem;
+    &:first-child {
+      margin-left: 1.2rem;
+    }
+    &:last-child {
+      margin-right: 1.2rem;
+    }
+  }
+  .input {
+    width: 100%;
+    font-family: Lato;
+    font-style: normal;
+    font-weight: normal;
+    padding: 8px 12px;
+    //hack but fixes chrome default styling
+    &[type="date"],
+    &[type="time"] {
+      padding: 7px 12px;
+    }
+    font-size: 1.6rem;
+    line-height: 2.4rem;
+    color: $gray-4;
+    outline-width: 0;
+    border: unset;
+    border-radius: 0.8rem;
+  }
+
+  &.hover {
+    border: 1px solid $brand-blue-4;
+    box-shadow: 0px 0px 4px rgba(0, 109, 199, 0.4);
+  }
+
+  &.focus {
+    border: 1px solid $brand-blue-6;
+    box-shadow: 0px 0px 4px rgba(0, 109, 199, 0.4);
+    svg {
+      color: $gray-8;
+    }
+  }
+  // this setts the collors for icons and input
+  .input[type="text"] {
+    color: $gray-8;
```

Comment:
I'm not sure why we're setting `$gray-4` on line 42 and `gray-8` here. Can we move this up  to that same code block and just do the `&[]` like we do on line 36? For consistency. Also, what is this `gray-8` doing here? Isn't it only changing the color of text inputs? Then dates/pass, etc are all different? I could be wrong I'd like to hear more on this